submissions/pseudoscience/pseudoscience_non-fiction.md

```diff
@@ -0,0 +1,29 @@
+## A scientific examination of the Factorio world
+
+Suspension of disbelief is a well-known concept, however that does not stop me from ignoring it completely. Some days ago I saw a [Reddit post](https://www.reddit.com/r/factorio/comments/jaz6yl/calculating_the_density_of_nauvis/) about calculating the density of Nauvis, and that gave me an idea of making a just-for-fun compilation of all the ridiculous science that goes on in the Factorio universe. I will not go into the formulas because that would make this article sound like a research paper. Instead, I leave the exact calculations as an exercise for the especially dedicated reader.
+
+### Nauvis is a white dwarf
+
+This is a short recap of the Reddit post by [u/DaveMcW](https://www.reddit.com/user/DaveMcW/) that I just mentioned. We know the length of one day/night cycle on the planet, which gives us its angular velocity. By stating that the centripetal force from the planet's rotation has to be less than gravity at the equator and then cancelling a bunch of values, we get that Nauvis's density has to be greater than 813400 kg/m\^3. That makes Nauvis at least as dense as a white dwarf star.
```

Comment:
Could you talk about what the consequences of this and what white dwarf stars are, for those unfamiliar with star physics? Not too deep but a bit of insight could be nice. Also maybe a comparison of what Nauvis supposedly looks like vs what a white dwarf star looks like using a couple images to really illustrate the point? Not entirely sure where this image would come from though...